requirements-conda.txt

```diff
@@ -23,13 +23,6 @@ vphaser2=2.0
 biopython=1.68
 matplotlib=1.5.3
 future>=0.15.2
-PyYAML=3.11
 pysam=0.9.1
-pycodestyle
-mock==2.0.0
-six<2
-pytest=3.0.5
```

Comment:
I was trying to see if there was any significant negative impact on build time from moving these packages from conda to normal pip install, but it doesn't seem like it: the pip install seems to take < 20 seconds anyway. That said, it was kind of nice that folks could previously set up a dev environment by using conda only (and not having to pip install anything). We could add a note about it, but we probably need to improve the "how to set up your dev environment" documentation anyway, so we could leave that for a separate PR later on.